sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -194,6 +194,30 @@ public PipelineMessage CreateMessage()
         return message;
     }
 
+    /// <summary>
+    /// Creates a <see cref="PipelineMessage"/> that can be sent using this
+    /// pipeline instance with the specified URI, HTTP method, and response
+    /// classifier.
+    /// </summary>
+    /// <param name="uri">The <see cref="Uri"/> for the HTTP request.</param>
+    /// <param name="method">The HTTP method for the request.</param>
+    /// <param name="classifier">The <see cref="PipelineMessageClassifier"/>
+    /// to use for determining response success and retry behavior. If not provided,
+    /// <see cref="PipelineMessageClassifier.Default"/> will be used.</param>
+    /// <returns>The created <see cref="PipelineMessage"/> with the specified
+    /// URI, method, and classifier configured.</returns>
+    public PipelineMessage CreateMessage(Uri uri, string method, PipelineMessageClassifier? classifier = default)
```

Comment:
Should we hide the other parameterless factory method? When would someone use that one over this new one?

Reply:
I am on the fence. Let's merge as is and play with the API and see how it feels. I have to say I sometimes get annoyed when a totally fine API that I was using suddenly disappears from IntelliSense.